Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -1776,10 +1776,10 @@ def gen2018HiMix(fragment,howMuch):
     '--eventcontent': 'RECOSIM,DQM',
 }
 step3_riemannFit = {
-    '--procModifiers': 'riemannFit',
+    '--procModifiers': 'brokenLine',
```

Comment:
I'm not sure we want to do this, i.e. overwrite the riemannFit modifier and use it for the Broke line case.
@makortel how much work would it be to add another .X workflow with the Brokenline modifier? I'm not familiar with the runTheMatrix business.

Reply:
We don't - indeed that was there only for testing, thanks for reminding me about it.

Adding a .X workflow is not that hard, but I would prefer to avoid the increase...